### PR TITLE
Address selector number on Tour model

### PR DESCRIPTION
**Pull request by zachgarwood**
The tour selector number was not appearing in the JSON data, so I fixed the issue and updated the test.

### CODE REVIEW

File: app/Models/Transformers/TourTransformer.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ public function transform(TwillModelContract $tour)
             'image_url' => $tour->image_url,
             'thumbnail_full_path' => $tour->thumbnail_full_path,
             'large_image_full_path' => $tour->large_image_full_path,
-            'selector_number' => (string) $tour->selector_number,
+            'selector_number' => (string) $tour->selector->number,
```
Comment (nikhiltri) [inline app/Models/Transformers/TourTransformer.php]: Add a catch in case `$tour->selector` is null.

File: tests/Feature/TourSerializerTest.php [hunk 2]
```diff
@@ -21,12 +25,15 @@ public function test_serialize(): void
...
             $this->assertArrayHasKey('latitude', $tour);
+            $this->assertIsString($tour['latitude']);
             $this->assertArrayHasKey('longitude', $tour);
+            $this->assertIsString($tour['longitude']);
             $this->assertArrayHasKey('floor', $tour);
             $this->assertArrayHasKey('image_url', $tour);
             $this->assertArrayHasKey('thumbnail_full_path', $tour);
             $this->assertArrayHasKey('large_image_full_path', $tour);
             $this->assertArrayHasKey('selector_number', $tour);
+            $this->assertIsString($tour['selector_number']);
```
Comment (nikhiltri) [inline tests/Feature/TourSerializerTest.php:36]: `selector_number`, `latitude` and `longitude` are strings and not numeric fields?

**Review by nikhiltri — CHANGES_REQUESTED** [review]

File: tests/Feature/TourSerializerTest.php (hunk 2 above)
Comment (zachgarwood) [reply tests/Feature/TourSerializerTest.php:36]: Yup, that's how they're typed in the JSON:
Screenshot 2023-09-29 at 10 41 06 AM

Comment (nikhiltri) [reply tests/Feature/TourSerializerTest.php:36]: Yikes, ok.

**Review by nikhiltri — APPROVED** [review]
Looks good! ⭐

**Pull request merged**